bin/rails

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
-  load File.expand_path("../spring", __FILE__)
```

Comment:
What do you mean? It was there since the initial commit.

A quick google taught me the rails/spring is what makes rails reload everything in place while in development mode. I don't want to lose that.